src/simularium/types.ts

```diff
@@ -15,6 +15,19 @@ export interface VisDataMessage {
     fileName: string;
 }
 
+interface Coordinates3d {
+    x: number;
+    y: number;
+    z: number;
+}
+
+export interface CameraTransform {
```

Comment:
I would call this something other than CameraTransform, because fov is part of a "projection transform" and the rest of the data is used for a "view transform".  I'd go for something like CameraSpec/CameraInfo/CameraData or even Camera(but that keyword tends to be already used in threejs etc)

Reply:
Changed to `CameraSpec` :)